### PR TITLE
DM-44244: Create rubintv-dev application and add it to Summit environment

### PR DESCRIPTION
**Pull request by sebastian-aranda**

### CODE REVIEW

File: applications/rubintv-dev/values-summit.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  siteTag: "summit"
+
+  imagePullSecrets:
+  - name: pull-secret
+
+  frontend:
+    debug: true
+    env:
+    - name: S3_ENDPOINT_URL
+      value: "https://s3.rubintv.cp.lsst.org"
```
Comment (mareuter) [inline applications/rubintv-dev/values-summit.yaml:11]: Tip: You can use some YAML tricks to not duplicate information that's used elsewhere in the file. See: https://github.com/lsst-sqre/phalanx/blob/main/applications/simonyitel/values-tucson-teststand.yaml at the `ccoods` and `mtaos` entries.

File: docs/applications/rubintv-dev/index.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
+.. px-app:: rubintv-dev
+
+#################################################################
+rubintv-dev — Real-time display front end development application
+#################################################################
+
```
Comment (mareuter) [inline docs/applications/rubintv-dev/index.rst:6]: Can we put a bit of text that flags this as a development app? I see there is something in the title, but a little more in the main body would be good.

**Review by mareuter — APPROVED** [review]
Some minor things to think about, but otherwise this looks good.

File: applications/rubintv-dev/values-summit.yaml (hunk 1 above)
Comment (sebastian-aranda) [reply applications/rubintv-dev/values-summit.yaml:11]: Nice!! I didn't know about this trick, thank you one more time Michael :pray:, I'll make the change.

**Pull request merged**